Tests/DatadogTests/Datadog/RUM/RUMMonitor/Scopes/RUMViewScopeTests.swift

```diff
@@ -56,10 +66,15 @@ class RUMViewScopeTests: XCTestCase {
             name: .mockRandom(),
             attributes: [:],
             customTimings: [:],
-            startTime: .mockAny()
+            startTime: .mockAny(),
+            dateCorrection: .zero
         )
 
-        _ = scope.process(command: RUMStartUserActionCommand.mockAny())
+        _ = scope.process(
+            command: RUMStartUserActionCommand.mockAny(),
+            context: context,
+            writer: writer
+        )
```

Comment:
After seeing this all over, I wonder if it wouldn't make sense to anticipate future needs by introducing container type bundling `context` and `writer` in RUMScope APIs. For example:
```swift
scope.process(command:rumScopeContext:)
```
where initially we will have:
```swift
internal struct RUMScopeContext {
   let sdkContext: DatadogV1Context
   let writer: Writer
}
```
I will keep door open for bundling other aspects (if necessary) without huge refactoring pain. WDYT @maxep ?